docker/docker-compose-local.yml

```diff
@@ -7,6 +7,7 @@ services:
       - "30333:30333"
       - "9933:9933"
       - "9944:9944"
+      - "9615:9615"
     image: chevdor/polkadot:latest
```

Comment:
would be great to refer to the latest official release here

Reply:
fixed